nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedFacetImpl.java

```diff
@@ -95,31 +94,33 @@ public Content get(final String path) {
   }
 
   @Override
+  @TransactionalStoreBlob
```

Comment:
This method doesn't save anything. Annotation should be on `upload(..)` method.

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/84#discussion_r343583193